src/mmrelay/meshtastic_utils.py

```
@@ -550,21 +550,54 @@ def connect_meshtastic(passed_config=None, force_connect=False):
+                                "Using forked meshtastic library with auto_reconnect support"
+                            )
+                        else:
+                            logger.debug("Using official meshtastic library")
+
+                        try:
+                            meshtastic_iface = meshtastic.ble_interface.BLEInterface(
+                                **ble_kwargs
+                            )
+                        except Exception as e:
+                            # BLEInterface constructor failed - this is a critical error
+                            logger.exception(f"BLE interface creation failed: {e}")
```

Ruff (0.14.8)

Redundant exception object included in `logging.exception` call

(TRY401)